scripts/run-pyflakes3

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+# This file is part of curtin. See LICENSE file for copyright and license info.
+
+pycheck_dirs=(
+    "console_conf/"
+    "subiquity/"
+    "subiquitycore/"
+    "tests/"
+)
+
+bin_files=(
+    "bin/console-conf-tui"
+    "bin/subiquity-tui"
+)
+CR="
+"
+[ "$1" = "-v" ] && { verbose="$1"; shift; } || verbose=""
+
+set -f
+if [ $# -eq 0 ]; then unset IFS
+   IFS="$CR"
+   files=( "${bin_files[@]}" "${pycheck_dirs[@]}" )
+   unset IFS
+else
+   files=( "$@" )
+fi
+
+cmd=( "python3" -m "pyflakes" "${files[@]}" )
+
+echo "Running: " "${cmd[@]}" 1>&2
+export PYFLAKES_BUILTINS="_"
```

Comment:
Ah hah! Thanks for this bit in particular :)

Reply:
Indeed!